lib/JiraClient.ts

```diff
@@ -0,0 +1,51 @@
+import fetch = require("node-fetch");
+
+const JIRA_DOMAIN = 'https://sonarsource-sandbox-608.atlassian.net';
+
+export class JiraClient {
+
+    constructor(private readonly jiraToken: string) { }
+
+    public async listTransitions(issueId: string): Promise<any> {
```

Comment:
Let's turn this into `findTransition`, so `listTransition` will disappear completely